src/main/java/org/jenkinsci/plugins/mesos/MesosSlaveInfo.java

```diff
@@ -460,7 +460,7 @@ public String getDisplayName() {
         private final boolean dockerPrivilegedMode;
         private final boolean dockerForcePullImage;
         private final boolean dockerImageCustomizable;
-        private final boolean isDind;
```

Comment:
Are we sure that this is broken? From my understanding, "null" primitive values default to whatever value is specified by 0; For boolean, this is value `false.`

Reply:
I thought the same. @kaiwalyajoshi wanted to test this.